INN/umbrella-inndev
# Homepage Customizer improvements, and other items

## Changes

This pull request makes the following changes:

- https://github.com/INN/inn/pull/127
- on an INN Member List widget, this defines `$instance['title']` in the widget form if it is not already set in the instance settings. This was causing a PHP notice:

> PHP Notice:  Undefined index: title in /wp-content/plugins/inn-members/inn-members.php on line 247

## Why



For https://github.com/INN/inn/issues/125

## Testing/Questions

Features that this PR affects:

- homepage
- a widget that does not appear to be in use


Questions that need to be answered before merging:


Steps to test the widget portion of this PR:

0. Watch your site's debug log with WP_DEBUG enabled
1. Add an "INN Member List" to a widget area, and watch the log. Save it. Change its name. Save it. There should be no log output as a result.
2. Open the Customizer. There should be no log output as a result.
